backtester/report/report_test.go

```diff
@@ -27,16 +25,7 @@ func TestGenerateReport(t *testing.T) {
 	e := testExchange
 	a := asset.Spot
 	p := currency.NewPair(currency.BTC, currency.USDT)
-	tempDir, err := ioutil.TempDir("", "")
-	if err != nil {
-		t.Fatalf("Problem creating temp dir at %s: %s\n", tempDir, err)
-	}
-	defer func(path string) {
-		err = os.RemoveAll(path)
-		if err != nil {
-			t.Error(err)
-		}
-	}(tempDir)
+	tempDir := t.TempDir()
 	d := Data{
 		Config:       &config.Config{},
 		OutputPath:   filepath.Join("..", "results"),
```

Comment:
Can change this to `t.TempDir()` and remove the above declaration. Seems like we are overriding this anyway? 🤷

Reply:
@Juneezee Thank you!